tests/cpp/test_multidevice_sharding.cpp

```diff
@@ -106,6 +106,87 @@ TEST_P(MultideviceShardingTest, ShardGlobalInput) {
       runtime.completeFusion(), outputs, inputs, {x1, x2}, __LINE__, __FILE__);
 }
 
+TEST_F(MultideviceShardingTest, Slice) {
```

Comment:
These tests are a bit overkill... Slice introduces non-tv ops for indexing and normalization is a multi-output expression. 
They will also get tested in the transformer attention unit test when it can be merged, which would contain these expressions.